front/internals/webpack/webpack.config.js

```diff
@@ -102,27 +117,16 @@ const config = {
       },
       {
         test: /\.(svg|jpg|png|gif)$/,
-        loader: 'url-loader',
-        options: {
-          limit: 8192,
-        }
+        type: 'asset/resource',
```

Comment:
No need for url and file loader anymore, webpack 5 handles that out of the box!!! 🚀